pymongo/encryption_options.py

```diff
@@ -45,6 +45,8 @@ def __init__(
         mongocryptd_spawn_path: str = "mongocryptd",
         mongocryptd_spawn_args: Optional[List[str]] = None,
         kms_tls_options: Optional[Mapping[str, Any]] = None,
+        csfle_path: Optional[str] = None,
+        csfle_required: Optional[bool] = False,
```

Comment:
Remove Optional, just "bool" is fine. Can you do the same for bypass_auto_encryption?

Reply:
Done.